internal/characters/nahida/skill.go

```diff
@@ -0,0 +1,261 @@
+package nahida
+
+import (
+	"github.com/genshinsim/gcsim/internal/frames"
+	"github.com/genshinsim/gcsim/pkg/core/action"
+	"github.com/genshinsim/gcsim/pkg/core/attributes"
+	"github.com/genshinsim/gcsim/pkg/core/combat"
+	"github.com/genshinsim/gcsim/pkg/core/event"
+	"github.com/genshinsim/gcsim/pkg/core/glog"
+	"github.com/genshinsim/gcsim/pkg/enemy"
+)
+
+var skillPressFrames []int
+var skillHoldFrames []int
+
+func init() {
+	skillPressFrames = frames.InitAbilSlice(32)
+	skillPressFrames[action.ActionAttack] = 28
+	skillPressFrames[action.ActionCharge] = 28
+	skillPressFrames[action.ActionSkill] = 32
+	skillPressFrames[action.ActionBurst] = 32
+	skillPressFrames[action.ActionDash] = 27
+	skillPressFrames[action.ActionJump] = 26
+	skillPressFrames[action.ActionSwap] = 25
+
+	skillHoldFrames = frames.InitAbilSlice(63)
+	skillHoldFrames[action.ActionAttack] = 57
+	skillHoldFrames[action.ActionCharge] = 58
+	skillHoldFrames[action.ActionSkill] = 62
+	skillHoldFrames[action.ActionBurst] = 62
+	skillHoldFrames[action.ActionDash] = 59
+	skillHoldFrames[action.ActionJump] = 62
+	skillHoldFrames[action.ActionSwap] = 57
+}
+
+const (
+	skillPressCD        = 300
+	skillHoldCD         = 360
+	skillPressHitmark   = 13
+	skillMarkKey        = "nahida-tri-karma"
+	skillICDKey         = "nahida-tri-karma-icd"
+	triKarmaParticleICD = "nahida-tri-karma-particle-icd"
```

Comment:
Since we went through with the `"nahida-q"` change, I believe these also should be named akin to `"nahida-e"`